src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs

```diff
@@ -7,5 +7,5 @@
     Version = "2.0.0",
     Description = "Provides support for UI localization.",
     Category = "Internationalization",
-    Dependencies = new[] { "OrchardCore.Settings" }
+    Dependencies = new[] { "OrchardCore.Settings", "OrchardCore.Localization" }
```

Comment:
OrchardCore.Localization has a dependency on OrchardCore.Localization?

Reply:
So meta ;) It should be in ContentLocalization ... mentioned we haven't tested it, good catch.

Reply:
Good catch .. that should set in Content localization